garage/misc/instrument.py

```diff
@@ -348,6 +349,7 @@ def run_experiment(stub_method_call=None,
                    script="scripts/run_experiment.py",
                    python_command="python",
                    mode="local",
+                   git_target=None,
```

Comment:
these are called refs or references in git terminology

https://git-scm.com/book/en/v2/Git-Internals-Git-References

Reply:
The variables, methods and docstrings that contained target were changed to use ref or reference instead.